english/karaoke/karaoke/sing.md

```diff
@@ -1,3 +1,10 @@
 You break out into a beautiful song! (at least you think so)
 All your friends begin laughing at you because its karaoke and you realize
-you are not very good at singing!
\ No newline at end of file
+you are not very good at singing!
+
+You continue to sing anyway. The drunk audience breaks into a standing ovation
+on your courage.  Your eyes meet with the girl of your dreams.
+
+Do you:
+[propose to her](propose-to-her/propose-to-her.md)
+[puke on the mike](puke-on-the-mike/puke-on-the-mike.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.